screenpy/actions/tryto.py

```diff
@@ -0,0 +1,79 @@
+"""
+try/accept logic using screenplay pattern
+"""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from screenpy.pacing import the_narrator
+from screenpy.speech_tools import get_additive_description
+
+if TYPE_CHECKING:
+    from screenpy import Actor
+    from screenpy.protocols import Performable
+
+
+class TryTo:
+    """
+    TryTo perform followup action if the first one fails.
+
+    Examples::
+
+        the_actor.will(TryTo(DoAction()).otherwise(DoDifferentAction())
+
+        the_actor.will(
+            TryTo(CheckIfOnDomain(URL())).otherwise(Open.their_browser_on(URL())
+        )
+    """
+
+    Second: tuple[Performable, ...]
+
+    def perform_as(self, the_actor: Actor) -> None:
+        """perform a try/accept using the two provided actions"""
+        # logs the first attempt even if it fails
+        # try:
+        #     with the_narrator.mic_cable_kinked():
+        #         the_actor.will(*self.First)
+        # except AssertionError:
+        #     try:
+        #         the_actor.will(*self.Second)
+        #     except Exception as exc:
+        #         raise exc from None
+
+        # Since we are fully expecting assertion error to be raised we kink the
+        # cable to avoid the explanation; which only happens when the cable is not
+        # kinked
+
+        # logs the first attempt only if it succeeds.
```

Comment:
Are these comments all meant to be here? Some of these feel like they'd be better put in the class docstring.

Reply:
So I left this in partially as a conversation starter about which way the action should perform and how things get logged.

When I first created the action this was the version I wrote.  It quickly became apparent that I didn't want to log the failed action so I changed it to the version we have now.  But I worry that _my_ taste might not be the right thing for everyone else.  

As of now, if we were to fail the first action for reasons that might not make sense, we wouldn't log it. Is that a problem?

Reply:
Do you have any thoughts about how this logic works before I pull the commented code?